administrator/language/en-GB/en-GB.com_config.ini

```
@@ -93,6 +93,8 @@ COM_CONFIG_FIELD_FILTERS_CUSTOM_BLACK_LIST="Custom Blacklist"
 COM_CONFIG_FIELD_FILTERS_NO_HTML="No HTML"
 COM_CONFIG_FIELD_FILTERS_NO_FILTER="No Filtering"
 COM_CONFIG_FIELD_FILTERS_WHITE_LIST="Whitelist"
+COM_CONFIG_FIELD_FLOC_BLOCKER_DESC="Send a header to not support the tracking through the FLoC method proposed by Google."
+COM_CONFIG_FIELD_FLOC_BLOCKER_LABEL="Block Google FLoC"
```

```suggestion
COM_CONFIG_FIELD_FLOC_BLOCKER_LABEL="Block FLoC"
```
The FloC specification was published by the Web Platform Incubator Community Group. It is not a W3C Standard nor is it on the W3C Standards Track

It is not "Google FloC" - although its authors are Google.